less/index.less

```diff
@@ -53,7 +53,180 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+@box-color: #9099A2;
+
```

Comment:
Good work choosing your mixins.  You end up using display flex a lot so creating a mixin makes sense.  Also, you can use variables inside of you mixins if you didn't know.